clang/test/Driver/sycl-device-optimizations.cpp

```diff
@@ -30,9 +30,9 @@
 // CHECK-NO-SYCL-EARLY-OPTS: "-fno-sycl-early-optimizations"
 
 /// Check that Dead Parameter Elimination Optimization is enabled
-// RUN:   %clang -### -fsycl -fsycl-dead-args-optimization %s 2>&1 \
+// RUN:   %clang -### -fsycl %s 2>&1 \
 // RUN:   | FileCheck -check-prefix=CHECK-DAE %s
-// RUN:   %clang_cl -### -fsycl -fsycl-dead-args-optimization %s 2>&1 \
+// RUN:   %clang_cl -### -fsycl %s 2>&1 \
```

Comment:
Adding the CC1 call checks for the negative option may also be worth it, despite a few other tests rely on it.